configuration/admin.py

```diff
@@ -12,7 +12,14 @@ class ConfigurationAdmin(ModelAdmin):
             "widget": JSONEditorWidget(options={"modes": ["tree", "code"], "mode": "tree", "enableDrag": False})
         },
     }
-    search_fields = ("name",)
+    search_fields = ("name", "white_label__name")
```

Comment:
Does "white_label__name" have an extra underscore here?

Reply:
Yes, the syntax in Django for filtering by a nested field is to use a double underscore. This is saying to search with the `name` field on the `white_label` model.